### ISSUE TITLE
beet modify on year confirmation dialog output does not print new value

### ISSUE THREAD

**Issue by bartkl** [beetbox/beets#807]
**TL;DR**: When using `beet modify` to change the `year` value,  in the confirmation dialog it seems the new value being displayed is actually a print of the old one.

I'm changing the `year` tag of the (only) album called "Knee Deep" doing:

```
$ beet modify album:"Knee Deep" year=4000
Modifying 8 items.
WhoMadeWho - Knee Deep - There's an Answer
  year: 2011 -> 2011
WhoMadeWho - Knee Deep - Every Minute Alone
  year: 2011 -> 2011
WhoMadeWho - Knee Deep - Musketeer
  year: 2011 -> 2011
WhoMadeWho - Knee Deep - All That I Am
  year: 2011 -> 2011
WhoMadeWho - Knee Deep - Nothing Has Changed
  year: 2011 -> 2011
WhoMadeWho - Knee Deep - Two Feet Off Ground
  year: 2011 -> 2011
WhoMadeWho - Knee Deep - Checkers
  year: 2011 -> 2011
WhoMadeWho - Knee Deep - We're Alive, It's a Miracle
  year: 2011 -> 2011
Really modify, move and write tags (Y/n)? 
```

Notice that the new field value is not 4000 as I specified, but equal to the old one. However, I noticed in my audio player that the value was in fact updated to 4000. So, in this confirmation dialog it seems the new value being displayed is actually a print of the old one.

When I issued `beet modify album:"Knee Deep" album="XXX"` the output displays the correct values, so it seems to be the case only for the `year` tag.
